ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyOptimizer.java

```diff
@@ -437,7 +437,7 @@ private void transitionLayoutOnlyViewToNativeView(
 
   private static boolean isLayoutOnlyAndCollapsable(@Nullable ReactStylesDiffMap props) {
     if (props == null) {
-      return true;
+      return false;
```

Comment:
But we're setting collapsable=false on all those children though -- I don't understand why that wouldn't be enough.